mkdocs.yml

```diff
@@ -96,6 +100,19 @@ nav:
       - dc_send_webhook: 'functions/webhooks/send-webhook.md'
       - dc_update_webhook: 'functions/webhooks/update-webhook.md'
 
+  - Events:
+    - __on_system_message: 'events/system-message.md'
+    - __on_discord_message: 'events/discord-message.md'
+    # - __on_discord_message_edit: 'events/discord-message-edit.md'
```

Comment:
Feel free to include the missing pages, so i can close #50

Reply:
alright